### PR TITLE
831750-UG for validation support in MaskedTextBox

### PR DESCRIPTION
**Pull request by BhaskarSuresh22**
### Description ###

Added Events page for SfMaskedTextBox control.

### CODE REVIEW

File: winui/Masked-TextBox/MaskedTextBox_Images/winui_masked_textbox_validation.gif
Comment (Manivannan-E) [inline winui/Masked-TextBox/MaskedTextBox_Images/winui_masked_textbox_validation.gif:1]: @BhaskarSuresh22 Change the gif image

File: winui/Masked-TextBox/Events.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+description: Learn all about the events in the MaskedTextBox (SfMaskedTextBox) control.
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# Events in WinUI MaskedTextBox
+
+This section explains about the events exposed by the MaskedTextBox control.
+
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Events.md:13]: @BhaskarSuresh22 This section explains the events available in the WinUI MaskedTextBox control.

File: winui/Masked-TextBox/Events.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## ValueChanging
+
+The ValueChanging event occurs when the [Value](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfMaskedTextBox.html#Syncfusion_UI_Xaml_Editors_SfMaskedTextBox_Value) property is changing in the MaskedTextBox. This event receives two arguments namely `this` that handles SfMaskedTextBox and MaskedTextBoxValueChangingEventArgs as objects.
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Events.md:16]: @BhaskarSuresh22 Change the explanation as we discussed.

File: winui/Masked-TextBox/Events.md (hunk 1 above)
Comment (BhaskarSuresh22) [reply winui/Masked-TextBox/Events.md:13]: @Manivannan-E  Done the changes

File: winui/Masked-TextBox/Events.md (hunk 2 above)
Comment (BhaskarSuresh22) [reply winui/Masked-TextBox/Events.md]: @Manivannan-E Done the changes.

File: winui/Masked-TextBox/Events.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+title: Events in WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the events in the MaskedTextBox (SfMaskedTextBox) control.
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# Events in WinUI MaskedTextBox
+
+This section explains the events available in the WinUI MaskedTextBox control.
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Events.md:12]: @BhaskarSuresh22 
This section provides information about the events available in the WinUI MaskedTextBox control.

## ValueChanging Event
The ValueChanging event occurs when the Value property is about to change in the MaskedTextBox control. The MaskedTextBoxValueChangingEventArgs object provides the following properties:

NewValue: Gets the current value of the MaskedTextBox control.
OldValue: Gets the previous value of the MaskedTextBox control.
IsValid: Gets or sets a boolean value indicating whether the input is considered valid based on the mask completion.
Cancel: Gets or sets a value indicating whether the event should be canceled.

private void MaskedTextBox_ValueChanging(object sender, MaskedTextBoxValueChangingEventArgs e)
{
    // Access the new and old values
    string newValue = e.NewValue;
    string oldValue = e.OldValue;

    // Check the validity of the input
    bool isValid = e.IsValid;

    // Cancel the event if needed
    if (newValue == "1234")
    {
        e.Cancel = true;
    }

    // Perform additional actions based on the value changing
    // ...
}

File: winui/Masked-TextBox/Events.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+The MaskedTextBoxValueChangingEventArgs object contains the following properties:
+
+* NewValue: Returns the current value of the MaskedTextBox control.
+* OldValue: Returns the previous value of the MaskedTextBox control.
+* IsValid: Gets or sets a boolean value indicating whether the input is considered valid according to mask completion.
+* Cancel: Gets or sets a value indicating whether the event should be canceled.
+
+## ValueChanged
+
+The [ValueChanged](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfMaskedTextBox.html#Syncfusion_UI_Xaml_Editors_SfMaskedTextBox_ValueChanged) event occurs when the [Value](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfMaskedTextBox.html#Syncfusion_UI_Xaml_Editors_SfMaskedTextBox_Value) property is changed in the MaskedTextBox control.
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Events.md:27]: @BhaskarSuresh22 

## ValueChanged Event

The ValueChanged event occurs when the Value property is changed in the MaskedTextBox control. The MaskedTextBoxValueChangedEventArgs provides the following properties:

IsMaskCompleted: Gets a boolean value indicating whether all the required inputs for the mask are completed.
NewValue: Gets the current value of the MaskedTextBox control.
OldValue: Gets the previous value of the MaskedTextBox control.

private void MaskedTextBox_ValueChanged(object sender, MaskedTextBoxValueChangedEventArgs e)
{
    // Access the new and old values
    string newValue = e.NewValue;
    string oldValue = e.OldValue;

    // Check if the mask input is completed
    bool isMaskCompleted = e.IsMaskCompleted;

    // Perform actions based on the value change
    // ...
}

Comment (BhaskarSuresh22) [reply winui/Masked-TextBox/Events.md]: @Manivannan-E, Done the changes

File: winui/Masked-TextBox/Events.md (hunk 3 above)
Comment (BhaskarSuresh22) [reply winui/Masked-TextBox/Events.md]: @Manivannan-E, Done the changes.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**